bundles/org.openhab.binding.astro/src/test/java/org/openhab/binding/astro/TestUtils.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.astro;
+
+import java.util.Calendar;
+import java.util.GregorianCalendar;
+import java.util.TimeZone;
+
+/***
+ * A set of test utilities methods.
+ * 
+ * @author Witold Markowski

```

Comment:
Please add a short description for what has been changed or "Initial contribution" if your are the one who introduced a  new class.

```suggestion
 * @author Witold Markowski - Initial contribution
```

Reply:
Ok.